### DISCUSSION TITLE
How can I keep a relative simple track of the version for a part

### DISCUSSION THREAD

**Discussion by jesseyu1984918** [inventree/InvenTree#7670] [Q&A]
Whenever I have entered a new revision, it either will have to be a new part or it will just override the old one, is there a neat and simple way to keep a part but track it's revision?

**Comment by SchrodingersGat** [comment]
@jesseyu1984918 you can use the "revision" field to track a text-code for the revision type, if that is what you mean. You can also edit the "notes" field and place information here...
